### ISSUE TITLE
Long match arms are not wrapped

### ISSUE THREAD

**Issue by solidsnack** [rust-lang-nursery/rustfmt#1460]
Consider:

```rust
         let formatted_token = match self.0 {
            REORDER_BUFFER_CHANGE_INSERT => "insert",
            REORDER_BUFFER_CHANGE_UPDATE => "update",
            REORDER_BUFFER_CHANGE_DELETE => "delete",
            REORDER_BUFFER_CHANGE_INTERNAL_SNAPSHOT => "internal_snapshot",
            REORDER_BUFFER_CHANGE_INTERNAL_COMMAND_ID => "internal_command_id",
            REORDER_BUFFER_CHANGE_INTERNAL_TUPLECID => "internal_tuplecid",
            REORDER_BUFFER_CHANGE_INTERNAL_SPEC_INSERT =>
                "internal_spec_insert",
            REORDER_BUFFER_CHANGE_INTERNAL_SPEC_CONFIRM =>
                "internal_spec_confirm",
            _ => "reorder_something",
         }
```
This becomes:

```rust
         let formatted_token = match self.0 {
            REORDER_BUFFER_CHANGE_INSERT => "insert",
            REORDER_BUFFER_CHANGE_UPDATE => "update",
            REORDER_BUFFER_CHANGE_DELETE => "delete",
            REORDER_BUFFER_CHANGE_INTERNAL_SNAPSHOT => "internal_snapshot",
            REORDER_BUFFER_CHANGE_INTERNAL_COMMAND_ID => "internal_command_id",
            REORDER_BUFFER_CHANGE_INTERNAL_TUPLECID => "internal_tuplecid",
            REORDER_BUFFER_CHANGE_INTERNAL_SPEC_INSERT => "internal_spec_insert",
            REORDER_BUFFER_CHANGE_INTERNAL_SPEC_CONFIRM => "internal_spec_confirm",
            _ => "reorder_something",
         }
```

Which then fails :/

**Comment by nrc** [comment]
Closed by #1488

**Comment by solidsnack** [comment]
🎉
🎂
